ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/commons/parameters/LoopFlowParameters.java

```diff
@@ -37,4 +41,21 @@ public double getLoopFlowViolationCost() {
     public double getLoopFlowConstraintAdjustmentCoefficient() {
         return loopFlowConstraintAdjustmentCoefficient;
     }
+
+    public static LoopFlowParameters buildFromRaoParameters(RaoParameters raoParameters) {
+
+        /*
+        for now, values of LoopFlowParameters are constant over all the SearchTreeRao
+        they can therefore be instantiated directly from a RaoParameters
+         */
```

Comment:
unnecessary

Reply:
the comment or the method?

Reply:
the comment. I think that if things ever change (ie parameters depend on perimeter), then we'll know we have to change this. no need to leave this comment. what do you think?

Reply:
yep I agree